compliance/tests/value_type_copy_behavior/value_type_copy_behavior.gs.ts

```diff
@@ -108,8 +108,8 @@ export function main(): void {
 	// === Value-Type Copy Behavior ===
 	// Assigning a struct (value type) creates independent copies.
 	// valueCopy1 and valueCopy2 get their own copies of 'original's data.
-	let valueCopy1 = original!.value.clone()
-	let valueCopy2 = original!.value.clone()
```

Comment:
Review the git diff and figure out why the boxing analysis is wrong here. Review design/BOXING_POINTERS.md for details. This output should not change due to this pr. Fix the transpiler.